### DISCUSSION TITLE
Node.js 12 actions are deprecated

### DISCUSSION THREAD

**Discussion by Shamil-Isayev** [aws-actions/configure-aws-credentials#628] [General]
Hello,

I would like to know when you are going to upgrade it from 12 to 16. Thank you in advance.

**Comment by Shamil-Isayev** [comment]
I found the answer to my question. Sorry, I couldn't find how to close this discuss.

**Comment by peterwoodworth** [comment]
For anyone who may happen to stumble upon this in the future and not know the answer, please use `v2` of this action
